src/components/ebay-combobox/combobox.stories.js

```diff
@@ -48,7 +48,7 @@ export default {
             description:
                 "default is `none`; available values are `none` or `list`. For list, will automatically filter results while typing.",
         },
-        "list-selection": {
+        listSelection: {
             control: { type: "text" },
```

Comment:
This is likely out of the scope for this issue, but this control type should probably be `type: "select"` since there are known values that can be set.